data/locale/id-ID.ini

```diff
@@ -0,0 +1,12 @@
+DroidCamOBS="DroidCam OBS"
+DeactivateWhenNotShowing="Nonaktifkan saat tidak ditampilkan"
+UseWiFi="Gunakan WiFi"
+Perangkat="Perangkat"
+Segarkan="Segarkan Daftar Perangkat"
+Resolusi = "Resolusi"
```

Comment:
* The keys (before '=' sign) need to stay the same, only the value (between "") must be changed.
* No spaces around =

Reply:
thanks for the correction